deps.edn

```diff
@@ -1,12 +1,12 @@
 {:paths ["src"]
- :deps {org.clojure/clojurescript {:mvn/version "1.10.439"}}
+ :deps {}
  :aliases {:dev {:extra-deps {fipp/fipp {:mvn/version "0.6.21"}
                               org.clojure/clojure {:mvn/version "1.10.0"}
                               org.clojure/clojurescript {:mvn/version "1.10.439"}
                               org.clojure/test.check {:mvn/version "0.10.0-alpha3"}}}
            :test {:extra-paths ["test"]
                   :extra-deps {org.clojure/clojure {:mvn/version "1.10.0"}
-                               org.clojure/clojurescript {:mvn/version "1.10.439"}
+                               ;;org.clojure/clojurescript {:mvn/version "1.10.439"}
```

Comment:
This can be uncommented as needed or we can follow up with another patch which has an alias that only includes ClojureScript. In any case, `:test`, which is the test alias for the Clojure side should have this commented out to prove the ceremony around the `JSValue` stuff in `m.util` does what it's supposed to.